pulsar_metrics/utils.py

```diff
@@ -82,6 +82,8 @@ def compare_to_threshold(value: float, threshold: Union[list, Number], upper_bou
         status = value < threshold if upper_bound else threshold < value
     elif isinstance(threshold, list) and (len(set(threshold)) == 2) and all(isinstance(i, Number) for i in threshold):
         status = True if (min(threshold) < value < max(threshold)) else False
+    elif threshold is None:
```

Comment:
if  threshold is None:
	  return status;
Better to check this condition first, if it is none . no use of checking the if condition